src/main/java/com/linkedin/kmf/services/JettyService.java

```diff
@@ -25,16 +33,39 @@
   private final Server _jettyServer;
   private final int _port;
 
+  private class ConfigPublisherApp extends ResourceConfig {
```

Comment:
Would it be more intuitive (and possible) to publish the jolokia port as part of the jolokia config rather than the jetty config?

Reply:
My problem with that is the implicit dependency that would create between JolokiaService and JettyService. Given how services are initialized and configured, how do you see JettyService loading the configuration of JolokiaService?